ShellScript/Bash.sublime-syntax

```diff
@@ -182,9 +182,9 @@ contexts:
       scope: keyword.control.case.begin.shell
       set: [case-body, case-item, case-item-first-character, case-preamble]
     - match: \bcontinue{{keyword_boundary_end}}
-      scope: keyword.control.continue.shell
+      scope: keyword.control.continue.shell keyword.control.flow.continue.shell
     - match: \bbreak{{keyword_boundary_end}}
-      scope: keyword.control.break.shell
+      scope: keyword.control.break.shell keyword.control.flow.break.shell
```

Comment:
I agree with @keith-hall in that it seems strange to have these two scopes. I think we can make do with just your new `keyword.control.flow.*` scopes, no?

Reply:
If a minor breakage is acceptable then yes, but I would think that we should then look for similar scopes that can be changed accordingly at the same time and also decide if `keyword.control` implies flow control like `if` and `while` or if those should also move down to `keyword.control.flow.*` for consistency.

Again, my goal was to add some extra specific scopes without really affecting anything else rather than make any kind of decision.